README.md

```diff
@@ -95,6 +110,23 @@ What are the differences between calling and referencing a function? Please prov
 
 ```text
 
+var brittany = {
+  currentMood: sad,
+  describe: function() {
+    console.log("Brittany's current mood is happy.");
+      this.currentMood = "happy";
+  }
+};
+brittany.descibe();
+
+Calling a function causes the statements in the function body to be executed. The following is an example of calling a function. The above function is called when Brittany's mood is described.
```

Comment:
Awesome answer to this question! I hope your feeling happy today and not sad though :)